main.go

```diff
@@ -65,34 +65,37 @@ func main() {
 
 func generateReply(message string) string {
 	message = strings.ToLower(message)
-	check := Checks{
-		Stupid: containsStupidQuestion(message),
-		Smart:  containsSmartQuestion(message),
-	}
 
-	// Note: add switch in case there will be more checks in the future
-	switch check {
-	case Checks{Stupid: true, Smart: false}:
-		return "На рынке Хопа!"
-	case Checks{Stupid: false, Smart: true}:
-		return "Держи ссылку с адресом рынка Хопа, раз в гугле забанили:\nhttps://goo.gl/maps/aqN4rzapdDXvRJNW9"
-	case Checks{Stupid: true, Smart: true}:
+	// Note: add if-else statement in case there will be more checks in the future
+	if stupidHopaBazarQuestion(message) && smartHopaBazarQuestion(message) {
```

Comment:
In this case, transitioning from the `switch` to the `if` statement makes the code less readable.

Reply:
Also, you deleted everything associated with `Checks` structure in the code, but not the structure type itself.